test/server/cards/01_SOR/events/StrikeTrue.spec.ts

```diff
@@ -0,0 +1,76 @@
+describe('Strike True', function() {
+    integration(function(contextRef) {
+        describe('Strike True\'s ability', function() {
+            beforeEach(function () {
```

Comment:
Just noticed this - for cases where each test setup is used for only one test case we have a more compact way of doing the setup inside the test case. See Reinforcement Walker tests for an example.

Not required at all, just letting you know the option if you find it more convenient